config/configcheck/configcheck.go

```diff
@@ -98,8 +98,11 @@ func validateConfigDataType(t reflect.Type) error {
 		nf := t.NumField()
 		for i := 0; i < nf; i++ {
 			f := t.Field(i)
-			if err := checkStructFieldTags(f); err != nil {
-				errs = append(errs, err)
+			//TODO Need remove this check after discussion
```

Comment:
@tigrannajaryan How do we fix this ? Since the config is also a composite it keeps looping.